### PR TITLE
Workaround for RN >= 0.49 metro-bundler check for single string liter…

### PR DESCRIPTION
**Pull request by ashwinphatak**
…al argument to require() (#1342)

## What, How & Why?

Reference to the issue(s) addressed by this PR: # ???

## ☑️ ToDos

* [x] 📝 Changelog entry
* [ ] 🚦 Tests
* [x] 📝 Public documentation PR created or is not necessary
* [x] 💥 `Breaking` label has been applied or is not necessary

*If this PR adds or changes public API's:*
* [ ] typescript definitions file is updated
* [ ] jsdoc files updated
* [ ] Chrome debug API is updated if API is available on React Native

### CODE REVIEW

**Comment by blagoev** [comment]
although this fixes the issue. The #1332 got a point that we should not be fighting the RN but working with it. These workarounds and node requires work but maybe it's worth investigating how we can remove them since they are workarounds still. 

@ashwinphatak  got a good point that #1332 may introduce a breaking change, but isn't it worth investigating how we can have #1332 in and not introducing the breaking change at the same time.

in short this fix just fixes the workaround
